src/cli/templates.hh

```diff
@@ -1443,6 +1452,9 @@ maintainer = "Beep Boop Corp."
 ; The title of the app used in metadata files. This is NOT a window title. Can contain spaces and special characters. Defaults to name in a [build] section.
 title = "Beep Boop"
 
+; Builds an extension when set to "extension".
+type = "extension"
```

Comment:
should this be here?

Reply:
Do you think we should comment it or it's better to remove it completely?

Reply:
Comment it out and set it to an empty string (default)